### PR TITLE
Barbican monitoring and logging

### PR DESCRIPTION
**Pull request by wupeiran**
The changes cover:
1.remove redundant system dependencies
2.add monitoring for process check and api check
3.add logging

### CODE REVIEW

File: roles/barbican/defaults/main.yml [hunk 1]
```diff
@@ -29,35 +29,31 @@ barbican:
     upper_constraints: '{{ openstack_meta.upper_constraints }}'
     python_dependencies:
         - { name: PyMySQL }
-    python_post_dependencies:
-        - { name: uwsgi }
         - { name: python-memcached }
+    python_post_dependencies: 
+        - { name: uwsgi }
```
Comment (nirajdp76) [inline roles/barbican/defaults/main.yml]: move `uwsgi` to `python_dependencies` and set `python_post_dependencies: []`

**Comment by narengan** [comment]
Can we move the magnum and v7 changes to a different PR?

**Comment by wupeiran** [comment]
move the uwsgi to python dependencies instead of post dependencies.

**Comment by nirajdp76** [comment]
ok to test

**Pull request merged**